lib/addMethod/addMethodREST.js

```diff
@@ -29,11 +29,15 @@ module.exports = function (methodName, config, afterHeadersFunction) {
 		return;
 	}
 
+	// TCP keep-alive shorter than 350 sec to prevent AWS NAT idle timeout: https://repost.aws/knowledge-center/lambda-vpc-timeout
+	const keepAliveAgent = new require('http').Agent({ keepAlive: true, keepAliveMsecs: 10000 });
```

Comment:
```suggestion
	const keepAliveAgent = new require('http').Agent({ keepAlive: true, keepAliveMsecs: 350000 });
```
Shouldn't the `keepAliveMsecs` be 350s or less?

Reply:
It would have to be strictly less than 350 seconds, I think. So the most we can do is 349000. I did 10 seconds instead because it was the example on the README of needle. AWS example does 20 seconds. It says:

> To prevent the connection from dropping, you can turn on TCP keepalive with a value of less than 350 seconds to reset the idle timeout. TCP keepalive prevents the connection from being idle by sending packets to the front-end and backend connections every 20 seconds.

@lawrencej004 any particular preference for the TCP keep alive heartbeat interval?